FAQS.md

```diff
@@ -0,0 +1,54 @@
+**Markor - Frequently Asked Questions**
+
+**Markor - FAQ**
+
+This Frequently Asked Question will be continually updated to accommodate new information with respect to new updates and changes on the App and will be used inside the App to help users find answers to their questions.
+
+
+**What is Markor?**
+
+Markor is an Android application developed by Gregor Santner. It allows you to write, save and open files in markdown, plain and .txt formats. A note app developed for easy and improved note taking in markdown and .txt. 
+
+
+**What size is Markor and hope it isn't harmful to device?**
+
+Markor is a simple and lightweigh app that doesn't comsume much of your device space or RAM when installed, it is well diagonised for any harm it might cause your device and has been confirmed harm free.
+ 
+
+**Where can I download the Markor?**
+
+Markor can be downloaded from various app store like Google Play Store and on F-droid app store. 
+[Google Play Store](https://play.google.com/store/apps/details?id=net.gsantner.markor)
+[F-droid](https://f-droid.org/repo/net.gsantner.markor_20.apk)
+
+
+**Can I backup my notes from Markor?** 
+
+There isn't backup feature yet on the app, presently notes can only be saved and stored directly to a folder in the device's file manager.
```

Comment:
Files are stored as is in the documents folders. You can freely copy, move and pack the files to an archive and copy them to any device, using any filemanager. Markor is a offline, local document editor, not a cloud office :).

Reply:
But, when installing markor does it overwrite storage Documents/markor/todo.txt and QuickNote.md ?
Also,
What about import/export | backup/restore app settings (useful when switching devices)

Currently I am at 0.2.4 and can not install newer versions without uninstalling first because trying to install newer versions gives me,
![](https://i.imgur.com/Vju6fyZ.png)

Reply:
No, nothing gets ever overwritten or deleted by markor unless told todo so (manually). Reinstalling/Updating does not change any document.

what you see is a problem regarding certs. F-Droid apk and my/gplay apk are differently signed and one has to update to where it is from. You can always uninstall and reinstall the other, and will lose no document.

Reply:
What is LinkBox?
Is it a special format .txt file
Under settings/Files and folders,
![](https://i.imgur.com/8OT0lWD.png)
There is a mention of LinkBox.txt but file does not exist, nor can I find a way to create a _LinkBox_:
![](https://i.imgur.com/CQ3VGFz.png)

Reply:
linkbox is just a normal txt file containing links. It will get created when you share a link e.g. from your browser to Markor. There you have an option "LinkBox" which pastes the link into that file